### DISCUSSION TITLE
Hello, when trying to add image in discord embed builder and then executing it image isnt visible

### DISCUSSION THREAD

**Discussion by VVriter** [discord-jda/JDA#2267] [Questions and Help]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [X] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

### Expected Behaviour

1111

### Code Example for Reproduction Steps

```java
channel.sendMessageEmbeds(new EmbedBuilder().setColor(Color.magenta).setTitle("Duck!").setImage(object.get("url").toString()).build()).queue();
```

### Code for JDABuilder or DefaultShardManagerBuilder used

```java
JDA jda = JDABuilder
                .createDefault(tocken)
                .addEventListeners(new Main())
                .addEventListeners(new Animals())
                .addEventListeners(new Queue())
                .addEventListeners(new org.example.TBTT.Seen())
                .addEventListeners(new Stats())
                .addEventListeners(new UselessFact())
                .enableIntents(GatewayIntent.MESSAGE_CONTENT)
                .build();
```

### Exception or Error

```yesyes
No exceptions
```

**Comment by Andre601** [comment]
What is `object`?

Also, you sure it isn't Discord? They recently had issues with embedding links, which includes URLs for images.

**Comment by SpaSinghOut** [comment]
This setup works for me when uploading a file inside an embed:

`channel.sendMessageEmbeds(new EmbedBuilder()
    .setImage("attachment://filename.png").build())
    .addFiles(FileUpload.fromData(new File("filename.png")))
    .queue()`
